content/_index.md

```diff
@@ -25,9 +25,9 @@ Microsoft, Amazon, and Meta.
 
 ## News
 
-* 2023-04-06: "All Systems Go! 2023" - Video recordings released. <br />
+* 2024-09-26: "All Systems Go! 2024" - Video recordings released. <br />
   All Systems Go, the foundational user-space Linux conference, has published recordings of all talks.
-  Head over to https://www.youtube.com/playlist?list=PLWYdJViL9EioDNHn7xIqQJLyCayNPKeYf to check them out!
+  Head over to https://www.youtube.com/playlist?list=PLWYdJViL9EipIImmvuoGFAeS-lKeHH2DD to check them out!
```

Comment:
I think the old videos are still relevant. I think keeping 2y of history is OK. I'd really keep the old links hence, and just add the new ones to the top

Reply:
They are kept, there's a specific section that keeps all the links and a menu on the left-hand side, this is just for the main body of the homepage

Reply:
ie there's the separate https://uapi-group.org/docs/ page that is different from this one